pages/admin/request/create-renewal.tsx

```diff
@@ -233,6 +248,8 @@ export default function CreateRenewal() {
           physicianCity: doctorInformation.city,
           physicianPostalCode: doctorInformation.postalCode,
           ...additionalInformation,
+          usesAccessibleConvertedVan: additionalInformation.usesAccessibleConvertedVan,
+          requiresWiderParkingSpace: additionalInformation.requiresWiderParkingSpace,
```

Comment:
I believe these two fields should be included in `...additionalInformation` above

Reply:
See comment below about the same thing in `pages/admin/request/create-new.tsx`